README-Italian.md

```diff
@@ -1,110 +1,116 @@
+| Test                | Status                                                                                                                                                                                                                                                                  |
+| :------------------ | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Netlify build       | [![CI](https://github.com/alan-turing-institute/the-turing-way/workflows/CI/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3ACI+branch%3Amaster)                                                                            |
+| No Latin Phrases    | [![Check for Latin Phrases](https://github.com/alan-turing-institute/the-turing-way/workflows/Check%20for%20Latin%20Phrases/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3A%22Check+for+Latin+Phrases%22+branch%3Amaster) |
+| No Large Files      | [![TestFileSizes](https://github.com/alan-turing-institute/the-turing-way/workflows/TestFileSizes/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3ATestFileSizes+branch%3Amaster)                                           |
+| No "_Lorem Ipsum_"s | [![Check for Lorem Ipsums](https://github.com/alan-turing-institute/the-turing-way/workflows/Check%20for%20Lorem%20Ipsums/badge.svg)](https://github.com/alan-turing-institute/the-turing-way/actions?query=workflow%3A%22Check+for+Lorem+Ipsums%22+branch%3Amaster)    |
 
-_The Turing Way_ is a lightly opinionated guide to reproducible data science.
-You can read it here: <https://the-turing-way.netlify.com>
-You're currently viewing the project GitHub repository where all of the bits that make up the guide live, and where the process of writing/building the guide happens.
+_The Turing Way_ è una guida dettagliata all riproducibilià nella scienza dei dati (data science).
+Puoi leggerla qui: <https://the-turing-way.netlify.com>
+Stai attualmente esplorando il progetto nella sua repository GitHub dove sono conservati tutti i bit che compongono la guida, e dove avvienve il processo di lettura e creazione della guida stessa.
 
-Our goal is to provide all the information that researchers need at the start of their projects to ensure that they are easy to reproduce at the end.
+Il nostro intento è quello di fornire tutte le informazioni di cui i ricercatori hanno bisogno all'avvio dei loro progetti, in modo che questi siano facili da riprodurre una volta terminati.
```

Comment:
Albeit correct, this translation might lead to misunderstandings.
I would remove "una volta terminati", which preserves the meaning but makes it easier to understand.